packages/website/src/Events/Input/Messages/GeneralMessagesDoc.md

```diff
@@ -0,0 +1,65 @@
+## BaseMessage
```

Comment:
Maybe you don't need a documentation for the BaseMessage... Since this is the base class that other messages are inherited from. However, if a documentation for the base class is neccesary, then I don't need we should follow the same doc template. For example, a doc for the base class should explain the internal methods and how the customized class can inherit from it.